qa/update-tests/src/test/java/io/camunda/zeebe/test/RollingUpdateTest.java

```diff
@@ -330,6 +336,8 @@ private void configureBroker(final ZeebeBrokerNode<?> broker) {
         .withEnv("ZEEBE_BROKER_CLUSTER_MEMBERSHIP_PROBETIMEOUT", "1s")
         .withEnv("ZEEBE_BROKER_CLUSTER_MEMBERSHIP_FAILURETIMEOUT", "2s")
         .withEnv("ZEEBE_BROKER_CLUSTER_MEMBERSHIP_SUSPECTPROBES", "2")
+        // ensure we have an exporter present to test sharing exporter state across nodes
+        .withEnv("ZEEBE_BROKER_EXECUTIONMETRICSEXPORTERENABLED", "true")
```

Comment:
:D That was it? Great :laughing:

Reply:
It failed pretty reliably, since in the tests when we update a broker, we wait for it to rejoin and for its partitions to be healthy.